packages/app/src/settings/store/migrations/002-privacy-policy-accepted.migration.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+// AUTO DISABLED - fix me if you like :)
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+export const privacyPolicyAcceptedMigration = (store: any) => {
+  // AUTO DISABLED - fix me if you like :)
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+  store.set("privacyPolicyAccepted", false)
```

Comment:
can't this any be something else? This seems to me as Map ;D

Reply:
here you go ;)  36dbdf54421885b34686c9203c2855820e73ff27